queries.sql

```diff
@@ -1,3 +1,25 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Lunas';
+-- Find all animals whose name ends in "mon".
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+-- List the name of all animals born between 2016 and 2019.
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+-- List the name of all animals that are neutered and have less than 3 escape attempts.
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+
+-- List the date of birth of all animals named either "Agumon" or "Pikachu".
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+
+-- List name and escape attempts of animals that weigh more than 10.5kg.
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+-- Find all animals that are neutered.
+SELECT * FROM animals WHERE neutered = true;
+
+-- Find all animals not named Gabumon.
+SELECT * FROM animals WHERE name <> 'Gabumon';
+
+-- Find all animals with a weight between 10.4kg and 17.3kg (inclusive).
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```

Comment:
- [ ] Great work on querying the `animal` table to meet the project's requirements! 👏 However, I noticed that the link to the screenshot of your queries in the pull request summary is inaccessible because the content is marked as private. 😕 Below is a screenshot of what I encountered when trying to preview your screenshot:

<img width="1430" alt="Screenshot 2023-09-22 at 22 31 00" src="https://github.com/Errytagedesign/vet_clinic/assets/107192076/43262127-aefb-42e4-9708-1d6941dc9d16">

  
 - To streamline the review process and ensure clarity, I'd recommend that you embed the screenshot directly into your pull request summary, as specified in the project requirements. 📷 This will make it easier for reviewers (and everyone involved) to quickly understand and assess your work. 

happy coding! 😊🚀🍀